assets/app/scripts/controllers/deploymentConfig.js

```diff
@@ -53,6 +53,7 @@ angular.module('openshiftConsole')
       .get($routeParams.project)
       .then(_.spread(function(project, context) {
         $scope.project = project;
+        $scope.logContext = context;
```

Comment:
Yeah, it was called `logContext` because it was specifically for streaming logs. I'm good with changing all references to the more general name.